commit 5024f3fef98ab01d4b8d35728982cc38b10de53f
Author: Stephen Diehl <stephen.m.diehl@gmail.com>
Date:   Mon Oct 1 17:35:27 2018 +0100

    Add support for GHC 8.6 (#94)
    
    * Add support for GHC 8.6
    
    * Fix STM version bounds on pragma

diff --git a/protolude.cabal b/protolude.cabal
index 16b5463..f72a46f 100644
--- a/protolude.cabal
+++ b/protolude.cabal
@@ -65,16 +65,16 @@ library
     -fwarn-implicit-prelude
 
   build-depends:       
-    base                >= 4.6  && <4.12,
+    base                >= 4.6  && <4.13,
     array               >= 0.4  && <0.6,
     ghc-prim            >= 0.3  && <0.6,
     async               >= 2.0  && <2.3,
     deepseq             >= 1.3  && <1.5,
-    containers          >= 0.5  && <0.6,
+    containers          >= 0.5  && <0.7,
     hashable            >= 1.2  && <1.3,
     transformers        >= 0.2  && <0.6,
     text                >= 1.2  && <1.3,
-    stm                 >= 2.4  && <2.5,
+    stm                 >= 2.4  && <2.6,
     bytestring          >= 0.10 && <0.11,
     mtl                 >= 2.1  && <2.3,
     mtl-compat          >= 0.2  && <0.3,
diff --git a/src/Protolude.hs b/src/Protolude.hs
index dd3ea31..5376ca6 100644
--- a/src/Protolude.hs
+++ b/src/Protolude.hs
@@ -492,7 +492,7 @@ import qualified Control.Exception
 import Control.Monad.STM as X (
     STM
   , atomically
-#if ( __GLASGOW_HASKELL__ < 806 )
+#if !(MIN_VERSION_stm(2,5,0))
   , always
   , alwaysSucceeds
 #endif
commit 8c88bc7d64dd8cf520e591af06827c65b24e2cb5
Author: Andre Van Der Merwe <andre@andrevdm.com>
Date:   Mon Oct 1 21:20:06 2018 +0200

    GHC 8.6.1 support (#95)
    
    * GHC 8.6.1 support
    
     - Bounds changes for base, containers and stm
     - stack lts 12.* tests
     - works with stack 8.6.1 nightly
        stack build --resolver nightly --haddock --test --bench --no-run-benchmarks
     - works with cabal new-build
    
    * GHCVER-8.6.1

diff --git a/protolude.cabal b/protolude.cabal
index f72a46f..6b433e7 100644
--- a/protolude.cabal
+++ b/protolude.cabal
@@ -1,5 +1,5 @@
 name:                protolude
-version:             0.2.2
+version:             0.2.4
 synopsis:            A small prelude.
 description:         A sensible set of defaults for writing custom Preludes.
 homepage:            https://github.com/sdiehl/protolude
@@ -25,7 +25,8 @@ tested-with:
   GHC == 7.10.3,
   GHC == 8.0.1,
   GHC == 8.2.1,
-  GHC == 8.4.1
+  GHC == 8.4.1,
+  GHC == 8.6.1
 extra-source-files: README.md CHANGES.md
 
 Source-Repository head
diff --git a/protolude.cabal b/protolude.cabal
index 6b433e7..b822096 100644
--- a/protolude.cabal
+++ b/protolude.cabal
@@ -1,5 +1,5 @@
 name:                protolude
-version:             0.2.4
+version:             0.2.3
 synopsis:            A small prelude.
 description:         A sensible set of defaults for writing custom Preludes.
 homepage:            https://github.com/sdiehl/protolude

commit 8c3c133342ad4b80871fddc6223b58c717e2b37e
Author: Stephen Diehl <stephen.m.diehl@gmail.com>
Date:   Tue Oct 2 10:37:58 2018 +0100

    Fix for TypeInType before 8.6

diff --git a/src/Protolude/Base.hs b/src/Protolude/Base.hs
index f4cae80..ddc9301 100644
--- a/src/Protolude/Base.hs
+++ b/src/Protolude/Base.hs
@@ -123,9 +123,10 @@ import GHC.Records as X (
 #if ( __GLASGOW_HASKELL__ >= 800 )
 import Data.Kind as X (
   type Type
-#if ( __GLASGOW_HASKELL__ < 806 )
+#if ( __GLASGOW_HASKELL__ < 805 )
   , type (*)
 #endif
+  , type Type
   )
 #endif
 
